inputfiles/patches/crypto.kdl

```diff
@@ -0,0 +1,6 @@
+//https://github.com/microsoft/TypeScript/issues/46036
+
```

Comment:
```suggestion
// https://github.com/microsoft/TypeScript/issues/46036
```

Can we use 2-space tabs for consistency with the source code btw? (I know the jsonc files still have 4-space tabs though)